src/IssueViz.UnitTests/Editor/LocationTagging/IssueLocationStoreAggregatorTests.cs

```diff
@@ -235,6 +235,45 @@ public void Refresh_HasStores_AllStoresRefreshed()
             }
         }
 
+        [TestMethod]
+        public void RefreshOnBufferChanged_NullFilePaths_ArgumentNullException()
```

Comment:
Duplicates of the existing "Refresh" tests that check the composite forwards the calls correctly.